lib/registries/addon/overview/template.hbs

```diff
@@ -66,7 +66,7 @@
             </div>
         {{/let}}
     {{else}}
-        {{#if this.showMobileView}}
+        {{#if (and this.showMobileView (not this.registration.withdrawn))}}
```

Comment:
Could you also hide the `<RegistriesOverviewMenu>` higher in this file when the registration is withdrawn?